### PR TITLE
loading state for friends tabs + fix for header colors

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] This changes the background color behavior on profiles. Now the color is stuck to the top always
- [x] The search bar uses the white form
- [x] The back button on profiles is white
- [x] Correct loading state for loading up followers instead of showing it as zero

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready
cc: @cecileboucheron

File: shared/profile/user/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export type Props = {|
   username: string,
 |}
 
-const colorTypeToStyle = type => {
+export const colorTypeToStyle = (type: 'red' | 'green' | 'blue') => {
```
Comment (buoyad) [inline shared/profile/user/index.js]: `BackgroundColorType`

File: shared/profile/user/index.js [hunk 2]
```diff
@@ -299,21 +293,21 @@ class User extends React.Component<Props, State> {
         />
       )
     }
-    const loading = this.props.followersCount == null || this.props.followingCount == null
+    const loading = !this.props.followers || !this.props.following
```
Comment (buoyad) [inline shared/profile/user/index.js]: I see `Followers (null)` flash in during profile load.

File: shared/profile/user/index.js (hunk 1 above)
Comment (chrisnojima) [reply shared/profile/user/index.js]: actually not exported anymore so i don't need the type

File: shared/profile/user/index.js [hunk 3]
```diff
@@ -350,13 +344,20 @@ class User extends React.Component<Props, State> {
...
     let chunks = this.state.width ? chunk(friends, itemsInARow) : []
-    if (chunks.length === 0 && this.props.followingCount !== null && this.props.followingCount !== null) {
-      chunks.push({
-        text: this.state.selectedFollowing
-          ? `${this.props.userIsYou ? 'You are' : `${this.props.username} is`} not following anyone.`
-          : `${this.props.userIsYou ? 'You have' : `${this.props.username} has`} no followers.`,
-        type: 'noFriends',
-      })
+    if (chunks.length === 0) {
+      if (this.props.following && this.props.following) {
```
Comment (MarcoPolo) [inline shared/profile/user/index.js]: ?

Comment (chrisnojima) [reply shared/profile/user/index.js]: should be followers, copy and paste of the incorrect code above on line 353

**Pull request merged**